src/database/votePost.js

```diff
@@ -2,9 +2,17 @@ import { db } from "database";
 import firebase from "firebase";
 
 export const voteForOption = ({ optionName, postId, userId }) => {
+  const increment = firebase.firestore.FieldValue.increment(1);
```

Comment:
Best practice is to have this as a function
```suggestion
  const increment = () => firebase.firestore.FieldValue.increment(1);
```

Reply:
is the built in way not a function?

Reply:
done